src/signIn.js

```diff
@@ -28,8 +29,26 @@ const loginEmailPassword = async () => {
   }
 };
 
-function showErrorLogin(){
-    errorSpan.style = "display: block"
+function showErrorLogin() {
+  errorSpan.style = "display: block";
 }
 
 btnLogin.addEventListener("click", loginEmailPassword);
+
+const monitorAuthState = async () => {
+  onAuthStateChanged(auth, (user) => {
+    if (user) {
+      console.log(user);
```

Comment:
Please double check me, but I believe that This is not good. logging user data might not be the right thing.

Reply:
checked and change it - it was show all user details but without password.
change to user is login